db/migrate/20230922123246_add_spire_unique_indices.rb

```diff
@@ -1,6 +1,6 @@
 class AddSpireUniqueIndices < ActiveRecord::Migration[6.1]
   def change
-    add_index :passengers, :spire, unique: true
+    add_index :passengers, :spire
```

Comment:
I'm going to assume that no one has pulled this migration and run it since it hit `main`. Although if they have, it'll just show up as a confusing `schema.rb` difference.